backend/backend/src/main/java/de/aaaaaaah/velcom/backend/runner/single/TeleRunner.java

```diff
@@ -235,6 +244,17 @@ public void handleResults(GetResultReply resultReply) {
 			return;
 		}
 
+		synchronized (lastResults) {
+			// TODO: Find a sensible value. Currently it keeps 2 results around
```

Comment:
Doesn't this code keep 3 results?

Reply:
If it is >= 2 it will throw one away and then add another so it should keep only 2. But yea, maybe I should remove the last after I have added this result, then it might be easier to read.

Reply:
true